test/EtherFiOracle.t.sol

```diff
@@ -648,13 +648,33 @@ contract EtherFiOracleTest is TestSetup {
         _initReportBlockStamp(report);
 
         vm.prank(alice);
-        report.accruedRewards = 1 ether;
+        report.protocolAccruedRewards = 1 ether;
         consensusReached = etherFiOracleInstance.submitReport(report);
         assertFalse(consensusReached);
 
         vm.prank(chad);
-        report.accruedRewards = 1 ether;
+        report.protocolAccruedRewards = 1 ether;
         consensusReached = etherFiOracleInstance.submitReport(report);
         assertTrue(consensusReached); // succeeded
     }
+
+    function test_execute_task_treasury_payout() public {
+        vm.startPrank(owner);
+        etherFiOracleInstance.addCommitteeMember(chad);
+        etherFiOracleInstance.setQuorumSize(2);
+        liquidityPoolInstance.setTreasury(address(owner));
+        vm.stopPrank();
+
+        _moveClock(1024 + 2 * slotsPerEpoch);
+
+        IEtherFiOracle.OracleReport memory report = _emptyOracleReport();
+        _initReportBlockStamp(report);
+
+        // Assume that the accruedRewards must be 1 ether, all the time
+
+        // Alice submited the correct report 
+        vm.prank(alice);
+        report.protocolAccruedRewards = 90 ether;
+        report.protocolFees = 10 ether;
```

Comment:
@vvalecha519 wen test

Reply:
(Discussion with V)

protocolAccruedRewards = rebase eeth (so includes tnfts eth and bnft eth where bnft owned by liquidity pool)
protocolFees = ethfund eth rewards + treasury rewards + no operator rewards

if 100 eth is earned by etherfi validators 
lets z= fraction of bnft held by lp
protocolAccruedRewards = 90 * 29/32 + 3/32*90*(z)
protocolFees = 10 + 3/32*90 * (1-z)

protocolAccruedRewards + protocolFees = 100

Reply:
@vvalecha519 where is the test for this scenario?

Reply:
you can spin up 2 vals so that z = 1/2 and do the test.